File: src/main/houpo/moveZeroes/moveZeroes.c
void moveZeroes(int* nums, int numsSize) {
    int i=0, nowIndex = 0;
    for( i=0;i<numsSize;i++){
        if(nums[i] != 0){
            nums[nowIndex] = nums[i];
            nowIndex ++;
        }
    }
    for( i=nowIndex;i<numsSize;i++){
        nums[i] = 0;
    }
}

File: src/main/houpo/moveZeroes/moveZeroes.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 * https://leetcode.com/problems/move-zeroes/description/
 */
var moveZeroes = function(nums) {
    var nowIndex = 0;
    for(var i=0;i<nums.length;i++){
        if(nums[i] != 0){
            nums[nowIndex] = nums[i];
            nowIndex ++;
        }
    }
    for(var j=nowIndex;j<nums.length;j++){
        nums[j] = 0;
    }
};
